frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -176,9 +179,20 @@ export const personsLogic = kea<personsLogicType<PersonFilters, PersonLogicProps
                     action = parsedValue !== undefined ? 'updated' : 'removed'
                 }
 
-                actions.setPerson(person) // To update the UI immediately while the request is being processed
-                const response = await api.update(`api/person/${person.id}`, person)
-                actions.setPerson(response)
+                actions.setPerson({ ...person }) // To update the UI immediately while the request is being processed
+
+                try {
+                    await api.create(`api/person/update_property`, {
+                        key,
+                        new_value: newValue,
+                        main_distinct_id: person.distinct_ids[0],
```

Comment:
This is a weird assumption - why not person_id?

Reply:
Because we need to send an event

Reply:
See how the merge people code path works for example

Reply:
Person pages working off of distinct ids is a historical accident waiting for us to clean it up at some point. Could we get the distinct_id by doing a person lookup in the endpoint instead?

Reply:
That's doable, yes

Reply:
why should we do a lookup, isn't that more work?

Reply:
Actually I see what @macobo is talking about in the sense that we used to have the keys for person pages be distinct IDs, which was a problem. However, I believe `personsLogic` will always contain a distinct ids list so I would agree with @tiina303 - no need for an extra lookup